com.reprezen.swagedit/src/com/reprezen/swagedit/editor/SwaggerContentDescriber.java

```diff
@@ -13,6 +13,8 @@
 import com.reprezen.swagedit.core.editor.TextContentDescriber;
 
 public class SwaggerContentDescriber extends TextContentDescriber {
+	
+	public static final String CONTENT_TYPE_ID = "com.reprezen.swagedit.contenttype.swagger.yaml";
 
     @Override
     protected boolean isSupported(String content) {
```

Comment:
@tfesenko Maybe we should do a regex match to be more flexible regarding whitespace following the colons in the tests in `isSupported`?